test/util/cli_helpers.py

```diff
@@ -11,16 +11,16 @@
 def invoke_cli(test_runner, *args, exit_code=0):
     capture_stream = io.StringIO()
     cli_args = list(itertools.chain(*args))
-
     with test_runner.assertRaises(SystemExit) as sys_exit:
         with redirect_stdout(capture_stream):
             with patch.object(sys, 'argv', ["cli"] + cli_args):
                 Cli().run()
 
     test_runner.assertEqual(sys_exit.exception.code, exit_code,
-        "ERROR: CLI returned an unexpected error status code: '{}'".format(cli_args))
+                            "ERROR: CLI returned an unexpected error status code: '{}'".format(cli_args))
```

Comment:
to large indentation

Reply:
this in line with the `sys_exit.exception.code` above